vision-quickstart (1)/app/src/main/java/com/google/mlkit/vision/demo/textdetector/TextRecognitionProcessor.java

```diff
@@ -62,6 +64,19 @@ public void stop() {
     @Override
     protected void onSuccess(@NonNull Text text, @NonNull GraphicOverlay graphicOverlay) {
         Log.d(TAG, "On-device Text detection successful");
+        recogonizedTextList.add(text.getTextBlocks().get(0).getLines().get(0).getText());
+        if(recogonizedTextList.size()==depth)
+        {
+            Log.d("Noise testing", "Frame id: " + 0 + "  Frame text:" + recogonizedTextList.get(0));
+            Integer differentFrames = 0;
+            for(int i=1;i<depth;i++) {
+                Log.d("Noise testing", "Frame id: " + i + "  Frame text:" + recogonizedTextList.get(i));
+                if(recogonizedTextList.get(i).equals(recogonizedTextList.get(i-1)) == false)
```

Comment:
this is kind of overcounting the noise, as an a -> b -> a transition is counted as two different lines rather than one different.  A better method might be to use a Map<String, int> to count all of them, then find out how many are not in the most commonly occuring one.

However this is nitpicking, what you have here is fine for this purpose.